### DISCUSSION TITLE
Week 4 post

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#99] [General]
I played the last game linked, CodeCombat. It only let me complete a couple levels before requiring payment in order to move on, so I didn't play further. The first levels were very basic. It's great for beginners and children. I liked that it had a dropdown list of options to use that would make the avatar move/fight. 

I'm still working through the HTML Registration Form. I like HTML because it makes the most sense to me, but I don't like how little the program guides you through the process. When I have errors, it's still hard to know what I've done wrong. I'm still wishing I had more information and exact examples of what I'm supposed to do, especially when something new is introduced then utilized repeatedly. 

1. Instructional webpage that teaches a lesson on story elements (plot, characters, setting, theme, etc.). There may need to be obvious "click here" buttons to help guide young students through the progressive website. 
2. Create a website portfolio for writing and designs. Make this easily accessible to viewers. It should be obvious where to go/click to navigate through the website.
3. Create an informative web page about a herbal business that is starting up. Make sure things are more professional instead of child-like. Add a FAQ page?
4. A webpage that teaches a lesson on what technical communication is. It needs to be obvious what to do to progress in the lesson/website. Once the information is learned, what do users do next?

**Comment by metasean** [comment]
You've definitely got some really good project ideas!

---

(tangent: every time that I see your user name, I get a nice little chuckle)

---

> When I have errors, it's still hard to know what I've done wrong. I'm still wishing I had more information and exact examples of what I'm supposed to do, especially when something new is introduced then utilized repeatedly.

To a large degree, this is actually intentional.  Part of being able to code, is learning how to troubleshoot and find information.

fwiw: The FreeCodeCamp forum typically has a lot of relevant information, for each step.  There's also something of a cheatcode – if you've run "Check Your Code" 3 or more times, you'll be given a question mark button, and it will let you easily filter the forum to that particular step.

![Screenshot 2023-06-05 at 18-26-42 freeCodeCamp org](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/7552754/a16471c5-061d-47d8-aadb-75b78a06acbc)


![Screenshot 2023-06-05 at 18-33-07 freeCodeCamp org](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/7552754/c9ad2f2c-658f-47f2-ae7d-bc3040ca3810)

**Reply by Skye404** [reply]
Thanks, I'll look into this more. I didn't even realize the 404 joke until now. I just chose random numbers after my name. Ha!
